ui/pages/confirm-import-token/confirm-import-token.js

```diff
@@ -51,7 +51,7 @@ const ConfirmImportToken = () => {
           token_contract_address: pendingToken.address,
           token_decimal_precision: pendingToken.decimals,
           unlisted: pendingToken.unlisted,
-          source_connection_method: pendingToken.isCustom
+          source: pendingToken.isCustom
```

Comment:
I know this isn't really part of the changes you set out to make but it looks like `isCustom` is a boolean? I'm not quite sure what we want it to be if isCustom is false. But for now seems like we should do:

```
pendingToken.isCustom ? MetaMetricsTokenEventSource.Custom : ''
```